tests/unit/dataset/dataset_download_test.py

```diff
@@ -1394,9 +1394,6 @@ def test_public_dataset_registered_correct_attributes(tmp_path, dataset_definiti
     assert dataset.definition.mirrors == dataset_definition.mirrors
     assert dataset.definition.resources == dataset_definition.resources
     assert dataset.definition.experiment == dataset_definition.experiment
-    assert dataset.definition.filename_format == dataset_definition.filename_format
```

Comment:
accessing this attribute is deprecated, so I removed it here. the `filename_pattern` is already tested two lines above via `assert dataset.definition.resources == dataset_definition.resources` so it shouldn't matter.